.travis.yml

```diff
@@ -15,10 +16,12 @@ branches:
   except:
   - requires-io-master
 install:
-- pip install "setuptools>=18.0.0,<30.0.0" # pypy support
+- pip install "setuptools>=18.5.0,<30.0.0" # pypy support
```

Comment:
maybe we should get rid of the <30, don't remember why we've added the top limitation (not remembering seems like a good reason to drop it)